hotsos/core/plugins/system/system.py

```diff
@@ -128,6 +128,67 @@ def unattended_upgrades_enabled(self):
 
         return False
 
+    @cached_property
+    def ubuntu_pro_status(self):
+        """Parse and retrieve Ubuntu Pro status
+
+        Returns:
+            Dictionary: Ubuntu pro status as a dictionary, e.g.::
+            {
+                "services": {
+                    "esm-apps": {
+                        "entitled": "yes",
+                        "status": "enabled"
+                    }, /* ... */
+                },
+                "account": "ACME Corporation",
+                "subscription": "Ubuntu Pro (Apps-only) - Virtual",
+                "technical_support_level": "essential",
+                "valid_until": "Sat Jan  1 01:01:01 9999 +03"
+            }
+
+            None: `pro/ua` command not available
+        """
+
+        pro_status = CLIHelper().pro_status()
+        if not pro_status or len(pro_status) < 1:
+            return
+
+        header = re.compile(r"^SERVICE +ENTITLED +STATUS +DESCRIPTION")
+        service_status = re.compile(
+            r"^([a-zA-Z0-9-]+) +([a-zA-Z0-9-]+) +([a-zA-Z0-9-]+) +(.+)\n")
+        kv = re.compile(r"^([A-Za-z0-9 ]+): (.*)\n")
+
+        if not header.match(pro_status[0]):
+            # Unexpected `pro status` header format
```

Comment:
maybe use log.info() here so that we can debug

Reply:
done